packages/react-i18n/src/test/babel-plugin.test.ts

```diff
@@ -37,7 +37,8 @@ describe('babel-pluin-react-i18n', () => {
           id: 'MyComponent_${defaultHash}',
           fallback: _en,
           async translations(locale) {
-            return await import(/* webpackChunkName: "MyComponent_${defaultHash}-i18n", webpackMode: "lazy-once" */ \`./translations/$\{locale}.json\`);
+            const dictionary = await import(/* webpackChunkName: "MyComponent_${defaultHash}-i18n", webpackMode: "lazy-once" */ \`./translations/$\{locale}.json\`);
+            return dictionary.default;
```

Comment:
I think it may be safer to do `dictionary && dictionary.default` since there is not a 100% guarantee a module will be resolved totally correctly (and undefined is a valid value to return)

Reply:
That makes sense, especially since this is added to the code post-compile. Done!